workshop/content/exercises/42-deploy-petclinic.md

```diff
@@ -2,8 +2,13 @@
 
 *   We have a Kustomization that we can use to easily get it up and running
 
+Clone the repository for the `kustomization.yaml`.
+```execute-2
+git clone https://github.com/springone-tour-2021/docker-services.git
```

Comment:
Like @tybritten said this is not necessary

Reply:
In the lab from March we had them clone the original and look at the file I was trying to keep the `disect petclinic` part that does that and use the files at the same time. We can have them clone the original repo. I was thinking to put the original as a reference for those that want to repeat locally if we go with the fork.

Reply:
I think we should just stick to referencing the fork throughout the workshop